show/main.py

```diff
@@ -228,6 +228,35 @@ def alias(interfacename):
 
     click.echo(tabulate(body, header))
 
+# 'Neighbor MetaData' subcommand ("show interfaces neighbor")
+@interfaces.command()
+@click.argument('interfacename', required=False)
+def neighbor(interfacename):
+    """Show Interface Neighbor"""
+    neighborCmd = 'sonic-cfggen -d --var-json "DEVICE_NEIGHBOR"'
+    p1 = subprocess.Popen(neighborCmd, shell=True, stdout=subprocess.PIPE)
+    neighbor_dict = json.loads(p1.stdout.read())
+
+    neighborMetaDataCmd = 'sonic-cfggen -d --var-json "DEVICE_NEIGHBOR_METADATA"'
+    p2 = subprocess.Popen(neighborMetaDataCmd, shell=True, stdout=subprocess.PIPE)
+    neighborMetaData_dict = json.loads(p2.stdout.read())
+
+    #Swap Key and Value
+    tempDict= {}
+    for port in natsorted(neighbor_dict.keys()):
+        tempDict[neighbor_dict[port]['name']]= {'localPort':port, 'neighborPort':neighbor_dict[port]['port']}
+
+    header = ['LocalPort','Neighbor','NeighborPort','NeighborLoopback','NeighborMgmt','NeighborType']
+    body = []
+    if interfacename is not None:
+        for device in natsorted(neighborMetaData_dict.keys()):
+            if tempDict[device]['localPort']==interfacename:
+                body.append([ tempDict[device]['localPort'],device, tempDict[device]['neighborPort'], neighborMetaData_dict[device]['lo_addr'],neighborMetaData_dict[device]['mgmt_addr'],neighborMetaData_dict[device]['type'] ])
+    else:
+        for device in natsorted(neighborMetaData_dict.keys()):
+            body.append([ tempDict[device]['localPort'],device, tempDict[device]['neighborPort'], neighborMetaData_dict[device]['lo_addr'],neighborMetaData_dict[device]['mgmt_addr'],neighborMetaData_dict[device]['type'] ])
```

Comment:
Please ensure one space after each comma.

Also remove space after `[` and before `]`

Consider breaking long line.